docs/features/connections.md

```diff
@@ -1,19 +1,22 @@
 ---
-id: connections 
-title: 'Grow your network with our connections feature!'
-sidebar_label: 'Connections'
+id: connections
+title: 'Grow Your Network with Our Connections Feature!'
+sidebar_label: Connections
 keywords:
-  - connections 
+  - connections
 ---
+
 ## What are Connections?
-Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our "List" feature, and you come across several promising developers. What's next? With Connections, you can now send a request to connect, bringing these experts into your professional circle for easy communication and collaboration.
+
+Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our [List](lists.md) feature, and you come across several promising developers. What's next? With Connections, you can now send a request to connect, bringing these experts into your professional circle for easy communication and collaboration.
 
 ![connections demo](../../static/gif/connections.gif)
 
 ## How Do Connections Work?
+
 - **Sending Connection Requests**: Once you identify users you'd like to connect with, you can send them a connection request.
 - **Acceptance**: Upon acceptance of your connection request, both parties become 1st-degree connections.
 - **Following Highlights**: After becoming 1st-degree connections, you can follow each other's highlights, keeping up-to-date with accomplishments, contributions, and other notable activities.
 - **Contact Information**: Post-acceptance, you'll have access to contact information to foster direct communication and ongoing dialogue.
 
-Excited? Login to [OpenSauced](https://opensauced.pizza/), create your Pro account or use a coupon to get this feature, and start building!😄
+Are you interested? Login to [OpenSauced](https://opensauced.pizza/), create your Pro account, or use a coupon to get this feature and start building!
```

Comment:
> ... or use a coupon to get this feature

If we want to mention about the coupon, we should as well mention how to get one.

Reply:
Let's remove the coupon reference.